free-programming-books.md

```diff
@@ -901,7 +901,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [The Bash Academy](http://guide.bash.academy) - Maarten Billemont (HTML)
 * [The Linux Command Line](http://linuxcommand.org/tlcl.php) - William E. Shotts, Jr. (PDF)
```

Comment:
You should move your book link to here(Between L902 and L903)
And you must leave 2 blank lines between last section and next heading,

Ref: [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) and https://github.com/EbookFoundation/free-programming-books/issues/3084#issue-391260916